src/pages/Login.js

```diff
@@ -81,8 +81,13 @@ export class Login extends Component {
                   </Form.Group>
                 )}
               </Field>
-              <Button variant="primary" type="submit" disabled={submitting}>
-                Login
+              <Button
+                className="button-animation"
+                variant="primary"
+                type="submit"
+                disabled={submitting}
+              >
+                <span>Login</span>
```

Comment:
I just noticed that all of these buttons have `<span>` wrapping the text - if we add that wrapper Button component that can be one thing that goes away.

Reply:
That would be awesome, I definitely think it's annoying to have to `<span>` the text inside every button